experimental/cloudstrapper/playbooks/roles/golang-host/vars/main.yaml

```diff
@@ -0,0 +1,8 @@
+---
+
+goVersion: 1.15.6
```

Comment:
Is there a way to always use the latest version ?

Reply:
I understand there are some dependencies we have with the go version our tools want to use. I will keep this for now.